diff --git a/node_modules/hive-multisig-sdk/dist/index.js b/node_modules/hive-multisig-sdk/dist/index.js
index dfa9a6d..594a782 100644
--- a/node_modules/hive-multisig-sdk/dist/index.js
+++ b/node_modules/hive-multisig-sdk/dist/index.js
@@ -116,6 +116,7 @@ class HiveMultisig {
                                 username: data.username,
                                 message: data.username,
                                 method: data.keyType,
+                                title: `Hive Multisig Login: verify ${data.keyType} key for @${data.username}`,
                             });
                             if (signBuffer.success) {
                                 let publicKey = signBuffer.publicKey
diff --git a/node_modules/hive-multisig-sdk/src/index.ts b/node_modules/hive-multisig-sdk/src/index.ts
index 3ec59bf..43c28ae 100644
--- a/node_modules/hive-multisig-sdk/src/index.ts
+++ b/node_modules/hive-multisig-sdk/src/index.ts
@@ -198,6 +198,7 @@ export class HiveMultisig {
               username: data.username,
               message: data.username,
               method: data.keyType,
+              title: `Hive Multisig Login: verify ${data.keyType} key for @${data.username}`,
             } as SignBuffer);
             if (signBuffer.success) {
               let publicKey = signBuffer.publicKey
